client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -27,20 +31,24 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
     const [isUnkonwnDateChecked, setIsUnkonwnDateChecked] = React.useState<boolean>(false);
     const [hasBackgroundDiseases, setHasBackgroundDiseases] = React.useState<boolean>(false);
     const [wasHospitalized, setWasHospitalized] = React.useState<boolean>(false);
-    const [isOtherSymptom, setOtherSymptom] = React.useState<string>('');
+    const [otherSymptom, setOtherSymptom] = React.useState<string>('');
     const [selectedSymptoms, setSelectedSymptoms] = React.useState<string[]>([]);
     const [selectedBackgroundDiseases, setSelectedBackgroundDiseases] = React.useState<string[]>([]);
     const [isOtherSymptomChecked, setIsOtherSymptomChecked] = React.useState<boolean>(false);
     const [isOtherBackgroundIllnessChecked, setIsOtherBackgroundIllnessChecked] = React.useState<boolean>(false);
     const [otherBackgroundIllness, setOtherBackgroundIllness] = React.useState<string>('');
+    const [isolationCityName, setIsolationCityName] = React.useState<string>('');
+    const [isolationStreetName, setIsolationStreetName] = React.useState<string>('');
+    const [streetsInCity, setStreetsInCity] = React.useState<Street[]>([]);
```

Comment:
It must be a state variable? Or maybe you can set it as a regular variable.

Reply:
why set as  a regular variable instead of state?

Reply:
Because you don't have to re-render the component if changes happen. You just need to keep it for the autocomplete component and send its value to update the address.

Reply:
streets are set in useClinicalDetails.tsx after axios request with city, and then state is used in clinicalDetails.tsx

Reply:
because the only streets i want saved are the streets in the chosen city